package.json

```diff
@@ -36,13 +36,15 @@
     "@snapshot-labs/prettier-config": "^0.1.0-beta.7",
     "@types/bn.js": "^5.1.0",
     "@types/jest": "^27.4.1",
-    "@types/mysql": "^2.15.21",
     "@types/node": "^18.11.6",
     "dotenv": "^16.0.1",
     "eslint": "^8.28.0",
     "jest": "^27.5.1",
     "jest-mock-extended": "^2.0.7",
+    "mysql": "^2.18.1",
+    "pg": "^8.10.0",
```

Comment:
Users will need to install DB client locally, we are installing those only as dev-dependencies to make those work when linking packages.